API.md

```diff
@@ -273,6 +279,7 @@ request.app['hapi-govuk-question-page'] = {
     isSpecialUser : request.auth.credentials.scope.includes('special')
   }
 }
+```
```

Comment:
missing closing tag was causing rendering issues in MD

Reply:
Good spot thank you!